HospitalSystem.Core/People/Patient/PatientService.cs

```diff
@@ -94,4 +94,33 @@ public Patient GetPatientById(ObjectId id)
     {
         return _repo.GetPatientById(id);
     }
+
+    public List<DateTime> GetAllTimesForMedicine (Prescription prescription)
+    {
+        DateTime now = DateTime.Now;
+        //TODO: unhardcode this
+        DateTime beginsToTake = new DateTime(now.Year, now.Month, now.Day, 8, 0, 0);
+        List<DateTime> times = new List<DateTime>();
+        for (int i=0; i<prescription.TimesADay; i++)
+        {
+            DateTime timeToTake = beginsToTake.AddHours(i*prescription.HoursBetweenIntakes);
+            times.Add(timeToTake);
+        }
+
+        return times;
+    }
```

Comment:
Does not belong in this service. Also could use a better name

Reply:
Any ideas?

Reply:
Should i make medicine notification service and place it there?

Reply:
VSCode is acting weird, i won't be able to test it...

Reply:
This is smart code. Maybe put in Prescription? Idk what that class looks like....